### PR TITLE
chore(ci): check GitHub Action workflow files via actionlint

### PR DESCRIPTION
**Pull request by orhun**
https://github.com/rhysd/actionlint

### CODE REVIEW

**Review by joshka — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -46,6 +46,10 @@ jobs:
         uses: crate-ci/typos@master
       - name: Lint dependencies
         uses: EmbarkStudios/cargo-deny-action@v1
+      - name: Check workflow files
+        uses: docker://rhysd/actionlint:latest
```
Comment (EdJoPaTo) [inline .github/workflows/ci.yml:50]: Not sure its really useful to keep this always running. CI is changing not that often. A link to the online check tool might be more relevant.

Comment (orhun) [reply .github/workflows/ci.yml:50]: I think it's fine, it's not much of a resource heavy thing anyways.

Comment (joshka) [reply .github/workflows/ci.yml:50]: Perhaps gate it on the path - not sure I like the docker approach to running something like this, but it seems like it takes 0 seconds to run, so not too much of a concern I guess.

Comment (orhun) [reply .github/workflows/ci.yml:50]: There are other ways of using this as well: <https://github.com/rhysd/actionlint/blob/main/docs/usage.md>

The docker approach looked safer than yolo-downloading a script and running it so I went with that

**Comment by orhun** [comment]
Going to close this one, it's something very trivial.

**Pull request closed without merging**